src/NUnitFramework/framework/Constraints/Comparers/DictionariesComparer.cs

```diff
@@ -54,10 +54,10 @@ internal DictionariesComparer(NUnitEqualityComparer equalityComparer)
                 return false;
 
             foreach (object key in xDictionary.Keys)
-                if (!_equalityComparer.AreEqual(xDictionary[key], yDictionary[key], ref tolerance))
+                if (!_equalityComparer.AreEqual(xDictionary[key], yDictionary[key], ref tolerance, false))
```

Comment:
Very minor, makes the code more readable IMO to use `topLevelComparison: false`.

Reply:
That works!  I didn't even know that was valid C# syntax... how long has that been a language feature?

Reply:
1.0 I think :D

Reply:
4.0 actually.